server/config.py

```diff
@@ -1,11 +1,26 @@
 import os
+import datetime
 
+# Env variables
 INNER_PORT = os.getenv("INNER_PORT")
 POSTGRES_DB_NAME = os.getenv("POSTGRES_DB_NAME")
 POSTGRES_USERNAME = os.getenv("POSTGRES_USERNAME")
 POSTGRES_HOST = os.getenv("POSTGRES_HOST")
 POSTGRES_PORT = os.getenv("POSTGRES_PORT")
 POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD")
+DEBUG = os.getenv("DEBUG") != "false"
+
+CONFERENCE_EXPIRATION_TIME = (
+    datetime.timedelta(seconds=int(os.getenv("CONFERENCE_EXPIRATION_TIME")))
+    if not DEBUG
+    else datetime.timedelta(seconds=int(os.getenv("DEBUG_CONFERENCE_EXPIRATION_TIME")))
+)
+CONFERENCE_GC_RATE = (
+    datetime.timedelta(seconds=int(os.getenv("CONFERENCE_GC_RATE")))
+    if not DEBUG
+    else datetime.timedelta(seconds=int(os.getenv("DEBUG_CONFERENCE_GC_RATE")))
+)
+
 
```

Comment:
It sure would. But I'm hesitating to do this rn because I think that it would mess up the gitflow. Adding an AppSettings class is a very important feature that will, I assume, require some rewriting across all the repo where the configuration constants are accessed. It would be counterintuitive to implement such thing in this exact feature-branch and will probably cause a lot of headache when merging. Prove me wrong if I'm wrong because I may be wrong but if I'm not wrong then I'm either right or neither wrong nor right which is still not wrong

Reply:
You're Schrödingerly wrong. Let me specify why you are not wrong. Well, that's a trivial thing, you have a centralized configuration storage which never gives you a way to change little pieces of configuration management without affecting the whole system. It is uncool. If you used real DDD, you'd have never faced this kind of problem. But I think that you can finish the feature and then make a separate branch for adding the `AppSettings` class to your configuration logic and make all the other features branches merge the one with `AppSettings`. Just keep in mind that you're really simplifying your work by adding pydantic to your config. And yeah, if you don't want to end up in such an unpleasant situation again, make more abstractions over the system's components, that's gonna work :) So you're not wrong that you are not adding `AppSettings` right now, you are wrong that you chose this way of configuration management in the beggining.

Reply:
That's fair. We'll meet again on the other branch then!